x-pack/plugins/ml/public/application/aiops/change_point_detection.tsx

```diff
@@ -25,7 +25,7 @@ export const ChangePointDetectionPage: FC = () => {
   const { services } = useMlKibana();
 
   const context = useMlContext();
-  const dataView = context.currentDataView;
+  const dataView = context.selectedDataView;
```

Comment:
could you please avoid using `useMlContext`? I'm going to remove it in https://github.com/elastic/kibana/pull/159176. I reckon we should use all Kibana services via `useMlKibana`. WDYT? cc @jgowdyelastic